game-app/game-core/src/main/java/games/strategy/engine/posted/game/pbf/NodeBbTokenGenerator.java

```diff
@@ -110,9 +112,9 @@ private void checkUser(final Map<?, ?> jsonObject, final String username) {
   private Map<String, Object> queryUserInfo(final CloseableHttpClient client, final String username)
       throws IOException {
     final String encodedUsername = URLEncoder.encode(username, StandardCharsets.UTF_8);
-    final HttpGet post = new HttpGet(forumUrl + "/api/user/username/" + encodedUsername);
-    HttpProxy.addProxy(post);
-    try (CloseableHttpResponse response = client.execute(post)) {
+    final HttpGet get = new HttpGet(forumUrl + "/api/user/username/" + encodedUsername);
```

Comment:
Is the change from 'get' to 'post' the major fix, or perhaps just part of it?

Reply:
If you look closely you'll notice that I only changed the variable name here, because it was kinda misleading